src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -26,12 +28,21 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
     <ErrorPage.Content className="mt-5 text-center">
       <Row>
         <Col xs={12} lg={{ span: 10, offset: 1 }}>
-          <ErrorPage.Title />
-          <ErrorPage.Subtitle>
+          <img
+            src={ExecutiveEducation2UErrorIllustration}
+            className="executive-education-2u-error-illustration"
+            alt="Executive Education 2U Error Illustration"
+          />
+          <div className="executive-education-2u-error-heading">
+            <span className="executive-education-2u-error-heading-red">
+              We&apos;re sorry.
+            </span>
+            <span className="executive-education-2u-error-heading-black">
+              Something went wrong.
+            </span>
+          </div>
+          <p className="executive-education-2u-error-heading-details">
             {createExecutiveEducationFailureMessage(failureReason)}
```

Comment:
This will be the actual error message, correct?

Reply:
Yes, this maps the failure reason to the actual error message.